ai-machine-learning/thoth-s2i-advise/deploy.md

```diff
@@ -0,0 +1,31 @@
+Now that you are familiar with the configurations our s2i image offers, let's try deploying our sample app using Thoth's S2I build process. 
+
+Assuming you have followed the steps from before are logged in as an admin into `myproject` in the openshift cluster lets deploy - 
+
+``oc process -f https://raw.githubusercontent.com/<your-github-username>/s2i-example/log-thoth/openshift.yaml | oc apply -f -``
```

Comment:
Maybe remove `<your-github-username>` and keep it pointing to thoth-station? Users could use the link/example directly without a fork.

EDIT: oh, I see the note bellow - I would say its fine to keep pointing/using just thoth-station.

Reply:
Umm, so should I remove that?
I thought users might want to experiment with their forks, but might be too extensive from a tutorial POV

Reply:
IMHO the tutorial should walk a user through the repo and what's present in there. Once they go through it, they are skilled enough to work on the fork.

Reply:
Makes sense, I will remove it :)

Reply:
Or how about this - 
If you haven't forked the repo and made any changes, you can try deploy our version using - 

```
``oc process -f https://raw.githubusercontent.com/thoth-station/s2i-example/log-thoth/openshift.yaml | oc apply -f -``{{execute}}

Else if you wanna try with the changes you made on your fork, try this - 
``oc process -f https://raw.githubusercontent.com/<your-github-username>/s2i-example/log-thoth/openshift.yaml | oc apply -f -``{{copy}}

```